content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -448,16 +448,18 @@ A ServiceAccount for `kube-proxy` is created in the `kube-system` namespace; the
 
 #### DNS
 
-A ServiceAccount for `kube-dns` is created in the `kube-system` namespace.
+A ServiceAccount for `coredns` is created in the `kube-system` namespace.
 
 Deploy the kube-dns Deployment and Service:
```

Comment:
s/kube-dns/coredns/

Reply:
s/coredns/CoreDNS

Reply:
@rajansandeep so this has to remain `kube-dns` as well?